doc/string/split.rdoc

```diff
@@ -12,47 +12,59 @@ When +field_sep+ is <tt>$;</tt>:
   the split occurs just as if +field_sep+ were given as that string
   (see below).
 
-When +field_sep+ is <tt>' '</tt> and +limit+ is +0+ (its default value),
-the split occurs at each sequence of whitespace:
+When +field_sep+ is <tt>' '</tt> (a single space), the split occurs at each sequence of whitespace.
+Any leading sequences of whitespace are removed from the string before splitting:
 
-  'abc def ghi'.split(' ')         => ["abc", "def", "ghi"]
+  'abc def ghi'.split(' ')          # => ["abc", "def", "ghi"]
   "abc \n\tdef\t\n  ghi".split(' ') # => ["abc", "def", "ghi"]
-  'abc  def   ghi'.split(' ')      => ["abc", "def", "ghi"]
-  ''.split(' ')                    => []
-
-When +field_sep+ is a string different from <tt>' '</tt>
-and +limit+ is +0+,
-the split occurs at each occurrence of +field_sep+;
-trailing empty substrings are not returned:
-
-  'abracadabra'.split('ab')  => ["", "racad", "ra"]
-  'aaabcdaaa'.split('a')     => ["", "", "", "bcd"]
-  ''.split('a')              => []
-  '3.14159'.split('1')       => ["3.", "4", "59"]
+  'abc  def   ghi'.split(' ')       # => ["abc", "def", "ghi"]
+  ''.split(' ')                     # => []
+  '  abc  def'.split(' ')           # => ["abc", "def"]
+
+When +field_sep+ is <tt>''</tt> (an empty string), the string is split into indivisual characters.
```

Comment:
```suggestion
When +field_sep+ is <tt>''</tt> (an empty string), the string is split into individual characters.
```

Reply:
Thank you! I also found another typo and fixed them together.